src/common/Network/PacketDef/Ipcs.h

```diff
@@ -216,7 +216,9 @@ namespace Core::Network::Packets
 
     // Unknown IPC types that still need to be sent
     // TODO: figure all these out properly
+    // Quests related op codes
```

Comment:
are you sure? do you have any examples where retail sends these packets alongside quest updates